# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-systemd-shared.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

From 9736f634c8b61343be966114ce1c9eddaf0fa742 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <luca.boccassi@gmail.com>
Date: Wed, 17 Sep 2025 16:13:37 +0100
Subject: [PATCH] meson: fix link-udev-shared option

This doesn't work anymore, setting it to false still makes
udev link to libsystemd-shared, as an argument was mistakenly
dropped.

Follow-up for 6350d2dbd97746440b9c8303ddc140ffda568732
---
 src/udev/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/udev/meson.build b/src/udev/meson.build
index dc54d4e68878c..908bec8b1d329 100644
--- a/src/udev/meson.build
+++ b/src/udev/meson.build
@@ -151,6 +151,7 @@ udev_binaries_dict = [
                         include_directories('.', 'net'),
                 ],
                 'dependencies' : udev_dependencies,
+                'link_with' : udev_link_with,
                 'install_rpath' : udev_rpath,
                 'install_tag' : 'udev',
                 'extract' : udevadm_extract_sources,

